.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media
```

Comment:
There is an extra space before the `/` in the `media` directory entry. It should be `media/` instead of `media /` to ensure the directory is correctly ignored.